packages/components/menu/src/menu-button.stories.ts

```diff
@@ -169,51 +181,116 @@ export const WithAvatar: Story = {
 };
 
 export const All: Story = {
```

Comment:
It looks like the focus jumps to another menu button when I'm using a keyboard to open a menu, it only works fine when I'm opening the first menu. I think the focus should stay on the menu button which has opened the menu.
<img width="659" alt="image" src="https://github.com/user-attachments/assets/333a1517-31fe-464a-9313-c6a87d4d4c60">

Reply:
Fixed